pkgs/applications/misc/lswt/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromSourcehut, nixos, wayland }:
+
+stdenv.mkDerivation rec {
+  pname = "lswt";
+  version = "1.0.4";
+
+  buildInputs = [ wayland ];
+
+  src = fetchFromSourcehut {
+    owner = "~leon_plickat";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Orwa7sV56AeznEcq/Xj5qj4PALMxq0CI+ZnXuY4JYE0=";
+  };
+
+  makeFlags = [
+    "DESTDIR=${placeholder ''out''}"
+    "PREFIX="
+  ];
+
+  meta = with lib; {
+    description = "A command that lists Wayland toplevels";
+    homepage = "https://git.sr.ht/~leon_plickat/lswt";
+    license = licenses.gpl3;
```

Comment:
is this gpl3Only or gpl3Plus?

Reply:
https://git.sr.ht/~leon_plickat/lswt/tree/master/item/LICENSE

Reply:
no, like nixpkgs does not accept new derivations using gpl3; you have to choose either only or plus, depending on if they accept only gpl v3 or newer versions, respectively.

Reply:
@Leon-Plickat `gpl3Only` or `gpl3Plus`?